sig/steep.rbs

```diff
@@ -1,6 +1,13 @@
 module Steep
   VERSION: String
 
+  # `ui_logger` is a logger for user interaction messages
+  #
+  # The *main* process has the logger.
+  # The *worker* processes disables the logging through this.
+  #
+  def self.ui_logger: () -> (Logger & ActiveSupport::TaggedLogging)
+
```

Comment:
I’m not certain about juggling two loggers. Perhaps it is possible to refactor things to use just one?
* By setting worker processes’ log level to `fatal`?
  * A logger has several severity levels. How many does Steep utilize?
    * Ruby’s `warn` and `-W` is enough for my projects.
* With a singleton method `warn` on Steep or on the logger object?

Reply:
Steep uses most of the log levels. Setting `fatal` in workers doesn't work. Adding `warn` method may work, but I prefer having two logger, for now.

So, we can refactor later if we have any problem.